config/application.rb

```diff
@@ -15,7 +15,13 @@
 module Pupilfirst
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.1
+    config.load_defaults 7.0
+
+    config.active_support.cache_format_version = 7.0
+
+    config.action_dispatch.cookies_serializer = :hybrid
```

Comment:
I don't think we need to set `cookies_serializer = :hybrid` any more. It's been a while since we've switched to `:hybrid`, so there shouldn't be any active sessions that still rely on `:marshal` serialized cookies.